Makefile

```diff
@@ -618,10 +618,7 @@ update-dependencies: ## Uses go get -u to update all the dependencies while hold
 	$(GO) mod tidy
 
 vet: ## Run mattermost go vet specific checks
-	@if ! [ -x "$$(command -v $(GOBIN)/mattermost-govet)" ]; then \
-		echo "mattermost-govet is not installed. Please install it executing \"GO111MODULE=off GOBIN=$(PWD)/bin go get -u github.com/mattermost/mattermost-govet\""; \
-		exit 1; \
-	fi;
+	$(GO) get -modfile=go.tools.mod github.com/mattermost/mattermost-govet/v2@new
```

Comment:
Was the bump to v2 needed after all?

Reply:
Semantically I'd say so. If it'd work on v5 and v6 both, then I'd be okay, but it takes only a single package path. So effectively, it doesn't work on old codebases.